termfactory/code_eT_zhz.py

```diff
@@ -123,17 +124,17 @@ def _eT_zhz_einsum_electronic_components_lhs(t_list, dT, z_right, b_loop_flag=Fa
         return electronic_components
 
     # otherwise
+    else:  # pragma: vectorize_b_loop
+        # for each t term add 1 electronic label
+        electronic_components += ['a', ] * len(t_list)
 
-    # for each t term add 1 electronic label
-    electronic_components += ['a', ] * len(t_list)
+        # the dT term has 1 electronic components
```

Comment:
At first I thought there was an issue and you deleted some code but I think its just git incorrectly recognizing the line changes

Reply:
Ah okay this can be fixed by adding an empty line after the `else` statement